nss-day-cohort-41/nss-day-cohort-41.github.io
# added video integration

## Link to Ticket

TITLE [Link]()

## Description of Proposed Changes
Removed Lacey from cohort.json
Added a capstone key to cohort.json objects
Added a conditional button in main.js to link to the video referenced in cohort.json object.capstone
- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <feature_branch>
serve -p 3000
take a random youtube url and substitute that for the null found in the capstone key of a student object
reload the main page and see if the video button has appeared and opens up a new tab to the video when clicked

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code
